### PR TITLE
JFG 981 Enum for standard metrics.

### PR DESCRIPTION
**Pull request by SokolAndrey**

### CODE REVIEW

File: chassis/core/src/main/java/com/griddynamics/jagger/user/test/configurations/limits/auxiliary/MetricName.java [hunk 1]
```diff
@@ -0,0 +1,55 @@
+package com.griddynamics.jagger.user.test.configurations.limits.auxiliary;
+
+import com.griddynamics.jagger.util.StandardMetricsNamesUtil;
+
+/**
+ * Enum for standard metrics.
+ */
+public enum MetricName {
```
Comment (dlatnikov) [inline chassis/core/src/main/java/com/griddynamics/jagger/user/test/configurations/limits/auxiliary/MetricName.java]: propose to rename to JMetricName

File: chassis/core/src/main/java/com/griddynamics/jagger/user/test/configurations/limits/auxiliary/MetricName.java [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+
+    SUCCESS_RATE_OK,
+    SUCCESS_RATE_FAILS,
+    THROUGHPUT,
+    VIRTUAL_USERS,
+    STD_DEV_LATENCY,
+    DURATION,
+    AVG_LATENCY;
```
Comment (dlatnikov) [inline chassis/core/src/main/java/com/griddynamics/jagger/user/test/configurations/limits/auxiliary/MetricName.java]: what about monitoring metrics?

File: chassis/core/src/main/java/com/griddynamics/jagger/user/test/configurations/limits/JLimitVsRefValue.java [hunk 3]
```diff
@@ -21,6 +22,10 @@ public static Builder builder(String metricName, RefValue refValue) {
         return new Builder(metricName, refValue);
     }
 
+    public static Builder builder(MetricName metricName, RefValue refValue) {
```
Comment (dlatnikov) [inline chassis/core/src/main/java/com/griddynamics/jagger/user/test/configurations/limits/JLimitVsRefValue.java]: we need good example of limits usage in our archetype

File: chassis/core/src/main/java/com/griddynamics/jagger/user/test/configurations/limits/JLimit.java [hunk 4]
```diff
@@ -145,15 +145,15 @@ public Builder withOnlyLowerThresholds(LowWarnThresh lowWarnThresh, LowErrThresh
          * Set all limits.
          * Cannot be initialized more than once.
          *
+         * @param lowErrThresh  lower error threshold.
          * @param lowWarnThresh lower warning threshold.
-         * @param lowErrThresh   lower error threshold.
          * @param upWarnThresh  upper warning threshold.
          * @param upErrThresh   upper error threshold.
```
Comment (dlatnikov) [inline chassis/core/src/main/java/com/griddynamics/jagger/user/test/configurations/limits/JLimit.java]: please, mention that all thresholds are relative. in all limits methods

File: chassis/util/src/main/java/com/griddynamics/jagger/util/StandardMetricsNamesUtil.java [hunk 5]
```diff
@@ -71,6 +71,49 @@
     public static final String DURATION_OLD_ID = "duration-old";
     public static final String ITERATION_SAMPLES_OLD_ID = "samples-old";
 
+
+
+    // standard monitoring metric names
+    public static final String MON_CPULA_1 = "mon_cpula_1";
```
Comment (dlatnikov) [inline chassis/util/src/main/java/com/griddynamics/jagger/util/StandardMetricsNamesUtil.java]: they are defined in com.griddynamics.jagger.dbapi.parameter.DefaultMonitoringParameters

File: chassis/core/src/main/java/com/griddynamics/jagger/user/test/configurations/limits/auxiliary/JMetricName.java [hunk 6]
```diff
@@ -0,0 +1,214 @@
...
+     * Convert {@link JMetricName} to String.
+     *
+     * @return String value of current metric name.
+     */
+    public String transformToString() {
+        String name = null;
+        switch (this) {
+            case SUCCESS_RATE_OK:
+                name = StandardMetricsNamesUtil.SUCCESS_RATE_OK_ID;
+                break;
```
Comment (a-badaev) [inline chassis/core/src/main/java/com/griddynamics/jagger/user/test/configurations/limits/auxiliary/JMetricName.java]: you can return the value right away.
also, you can use a Map with JMetricName as key and corresponding values. and here just get that value by a key

**Pull request merged**